Cabal/doc/developing-packages.rst

```diff
@@ -1,83 +1,119 @@
 Quickstart
 ==========
 
-Lets assume we have created a project directory and already have a
-Haskell module or two.
+Starting from scratch, we're going to walk you through creating a simple
+Haskell project.
 
-Every project needs a name, we'll call this example "proglet".
+**TL;DR;** ``mkdir proglet && cd proglet && cabal init --non-interactive && cabal run``
```

Comment:
isn't `--non-interactive` default now?

Reply:
Yup, updated, and also explicitly added `proglet` as the argument to `cabal run`.